Makefile

```diff
@@ -18,9 +20,9 @@ STRIP_DEBUG_FLAG=--v86-strip-debug
 endif
 
 default: build/v86-debug.wasm
-all: build/v86_all.js build/libv86.js build/v86.wasm
+all: build/v86_all.js build/libv86.js build/v86.wasm build/xterm.js
 all-debug: build/libv86-debug.js build/v86-debug.wasm
-browser: build/v86_all.js
+browser: build/v86_all.js build/xterm.js
```

Comment:
I'd prefer if xterm.js stayed optional — the code should handle a 404 fine (but let me know if it doesn't).